install/setup-js-sdk-assets.sh

```diff
@@ -21,16 +21,29 @@ if [[ "${SETUP_JS_SDK_ASSETS:-}" == "1" ]]; then
   # We want to remove everything before the first '{'.
   loader_registry=$(echo "$loader_registry" | sed '0,/{/s/[^{]*//')
 
+  #  Sentry backend provides SDK versions from v4.x up to v8.x.
+  latest_js_v4=$(echo "$loader_registry" | $jq -r '.versions | reverse | map(select(.|any(.; startswith("4.")))) | .[0]')
+  latest_js_v5=$(echo "$loader_registry" | $jq -r '.versions | reverse | map(select(.|any(.; startswith("5.")))) | .[0]')
+  latest_js_v6=$(echo "$loader_registry" | $jq -r '.versions | reverse | map(select(.|any(.; startswith("6.")))) | .[0]')
   latest_js_v7=$(echo "$loader_registry" | $jq -r '.versions | reverse | map(select(.|any(.; startswith("7.")))) | .[0]')
   latest_js_v8=$(echo "$loader_registry" | $jq -r '.versions | reverse | map(select(.|any(.; startswith("8.")))) | .[0]')
 
-  echo "Found JS SDKs v${latest_js_v7} and v${latest_js_v8}, downloading from upstream.."
+  echo "Found JS SDKs: v${latest_js_v4}, v${latest_js_v5}, v${latest_js_v6}, v${latest_js_v7}, v${latest_js_v8}"
```

Comment:
I wonder what decides which versions are available, and how this script will be properly kept up to date with newer releases, as this does not do auto-discovery based on metadata 🙇

Reply:
There's one JSON file that's always populated on Docker build time on the `sentry` container. (In which tested here https://github.com/getsentry/sentry/blob/fda38c5d70c4ef8c7340cfd5978d97ccf25b6674/self-hosted/Dockerfile#L85)

Reply:
We can find the current versions at the time of installation from https://github.com/getsentry/sentry-release-registry/tree/master/packages/npm/%40sentry/browser